desc/backend.py

```diff
@@ -595,9 +663,21 @@ def scan(f, init, xs, length=None, reverse=False, unroll=1):
             ys.append(y)
         return carry, np.stack(ys)
 
-    def bincount(x, weights=None, minlength=None, length=None):
-        """Same as np.bincount but with a dummy parameter to match jnp.bincount API."""
-        return np.bincount(x, weights, minlength)
+    def bincount(x, weights=None, minlength=0, length=None):
```

Comment:
To use `jnp.bincount` in a jitable manner, the `length` argument needs to specified, so the API of `jnp.bincount` differs from `np.bincount`, and we need to resolve it by creating an alias. On the jax backend `bincount` is an alias for `jnp.bincount`. On the numpy backend, `bincount` is defined as above. Otherwise, when we write code in DESC with `jnp.bincount(... length=)`, this code would call `np.bincount(... length=)` if JAX is not installed and throw an error.